utilities/egov-pdf/src/routes/workOrder.js

```diff
@@ -46,7 +46,13 @@ router.post(
             var organisation = resOrg.data
             if (contract && contract.contracts && contract.contracts.length > 0 && organisation && organisation.organisations && organisation.organisations.length > 0) {
                 var pdfResponse;
-                var pdfkey = config.pdf.work_order_template;
+                console.log(requestinfo.RequestInfo.msgId.split("|")[1])
+                if (requestinfo.RequestInfo.msgId.split("|")[1] == "en_IN") {
```

Comment:
add a null check , if requestinfo or msgId or some attribute is missing it should not fail.
also remove console.log